pages/directory/index.tsx

```diff
@@ -84,16 +111,36 @@ const Directory: NextPage<{ users: ClientUser[]; totalPages: number; error: stri
         </Typography>
         <Box>
           {error && <Typography>{error}</Typography>}
-          <List data-cy="user-container">
+          <List data-cy="user-container" sx={{}}>
             {userData.map((user, i) => {
               return (
                 <ListItem
                   key={user.id}
-                  sx={{ height: '100px' }}
                   ref={i === userData.length - 1 ? lastCardRef : null}
                   id={i === userData.length - 1 ? 'last-card' : null}
                 >
-                  {user.discord_name}
+                  <Card sx={{ margin: '0 auto', maxWidth: '500px', width: '95vw', height: '20em' }}>
+                    <Box sx={{  padding: '2em' }}>
+                      <Box sx={{ display: 'flex', justifyContent: 'space-around', padding: '2em' }}>
+                        {user.discord_avatar ?
+                          <Avatar alt={user.discord_name} src={user.discord_avatar} />
+                          : <Person sx={{ fontSize: '3em', fill: 'white' }}/>}
+                        { getUserSocials(user).length < 1 ? undefined : <Box sx={{ display: 'flex' }}>
+                          {getUserSocials(user).map(social =>
+                            <Link href={social.url} target='_blank' sx= {{ padding: '0 4em' }}>
+                              {social.icon}
+                            </Link>)}
+                        </Box> }
+                      </Box>
+                      <Box sx={{ display: 'flex', flexDirection: 'column' }}>
+                        <Typography variant="body1" sx={{ textAlign: 'center' }}>{user.discord_name}</Typography>
+                        <Typography variant="body1"
+                          sx={{ textAlign: 'center', maxWidth: '90%', margin: '0 auto' }}>
+                          {getShortenedBio(user.bio) }
+                        </Typography>
+                      </Box>
+                    </Box>
+                  </Card>
```

Comment:
nit: several of style/formatting inconsistencies

Also, I'm a bit puzzled by the styling... should we be using `sx` or TailwindCSS classes? Seems like there's a split between the two methods amongst the current codebase.